local-sys-test/sys-test_provider.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_providers {
+    athenz = {
+      source = "yahoo/provider/athenz"
+      version = "x.x.x"
```

Comment:
Shouldn't we specify a minimum version (such as in the example here https://www.terraform.io/language/providers/requirements:
      version = "~> 1.0")?
  
If I remember correctly, there are breaking changes between the terraform versions